src/Core/Content/Product/SalesChannel/Listing/ProductListingFeaturesSubscriber.php

```diff
@@ -172,6 +172,12 @@ public function handleListingRequest(ProductListingCriteriaEvent $event): void
 
             $this->sortingService->handleRequest($event);
         }
+
+        if (!$isOnCategoryPage) {
+            /** @var FindologicService $findologicService */
+            $findologicService = $event->getContext()->getExtension('findologicService');
+            $findologicService->disable();
+        }
```

Comment:
Shouldn't there be a test similar to the one in [PR#154](https://github.com/findologic/plugin-shopware-6/pull/154/files)?

Reply:
Nevermind, just read the whole story description 😅